packages/wdio-cli/src/run.js

```diff
@@ -1,11 +1,32 @@
+import fs from 'fs'
+import path from 'path'
 import logger from 'wdio-logger'
 
 import Launcher from './launcher.js'
+import setup from './setup'
 
 const log = logger('wdio-cli:run')
 
-export default function run (wdioConf, params) {
+export default function run (params) {
     let stdinData = ''
+    const commands = fs.readdirSync(path.join(__dirname, 'commands')).map((file) => file.slice(0, -3))
```

Comment:
Maybe use `path.extname(file)` instead of `String.slice`?

Reply:
you mean `basename`?

Reply:
I will go with `path.parse(file).name`